src/controllers/dynakube/version/updater.go

```diff
@@ -143,6 +147,11 @@ func setImageIDWithDigest( //nolint:revive
 	return nil
 }
 
+// TODO: this should be in a more general place
```

Comment:
I'm open for suggestions where to put this function.

Reply:
I would like to rename it, cause append is something when we add to exiting, here we join or concat strings.

Reply:
> I'm open for suggestions where to put this function.
maaybe `src/registry` ?